default/AI/PriorityAI.py

```diff
@@ -259,32 +266,47 @@ def calculateColonisationPriority():
     # print "Number of Colonisable planets : " + str(numColonisablePlanetIDs)
     # print "Priority for colony ships : " + str(colonisationPriority)
 
-    if colonisationPriority < 1: return 1
+    if colonisationPriority < 1: return 0
     return colonisationPriority
 
 
 def calculateOutpostPriority():
     """calculates the demand for outpost ships by colonisable planets"""
-    baseOutpostCost = AIDependencies.OUTPOST_POD_COST
+    base_outpost_cost = AIDependencies.OUTPOST_POD_COST
+
+    enemies_sighted = foAI.foAIstate.misc.get('enemies_sighted',{})
+    galaxy_is_sparse = ColonisationAI.galaxy_is_sparse()
+    total_pp = fo.getEmpire().productionPoints
+    num_colonies = len( list(AIstate.popCtrIDs) )
+    # significant growth barrier for low aggression, negligible for high aggression
+    if num_colonies > colony_growth_barrier:
+        return 0.0
+    mil_prio = foAI.foAIstate.get_priority(EnumsAI.AIPriorityType.PRIORITY_PRODUCTION_MILITARY)
+    allotted_portion = ([[[0.6, 0.8],[0.3, 0.4]],[[0.8, 0.9],[0.3, 0.4]]][galaxy_is_sparse]
```

Comment:
That reads better, yes.  The particular numbers here are also the kind of thing I would like to get moved into an AI_config setting.